api/views.py

```diff
@@ -80,6 +80,21 @@ def one_arg_reverse(self, request):
         reversed_url = reverse(reverse_string, args = [argument])
         return HttpResponse(json.dumps({reverse_string: reversed_url}))
 
+    @list_route(methods=['GET'])
```

Comment:
This example demonstrates how we'll eventually use the multipart upload....if Swagger works with it...